docs/terminal/basics.md

```diff
@@ -164,6 +164,10 @@ This can be configured using the `setting(terminal.integrated.rightClickBehavior
 * `selectWord` - Select the word under the cursor and show the context menu.
 * `nothing` - Do nothing and pass event to terminal.
 
+### Column selection
+
+`kbstyle(Alt)` and left-click drag allows selecting a rectangle of text inside the terminal instead of te regular selection of a line.
```

Comment:
```suggestion
Press `kbstyle(Alt)` and left-click drag to select a rectangle of text inside the terminal instead of the regular selection of a line.
```